src/aluno/PerfilPage.tsx

```
@@ -17,7 +17,15 @@ import {
     BreadcrumbPage,
     BreadcrumbSeparator
 } from '@/components/ui/breadcrumb'
-import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card'
+import {
+    Pagination,
```

ESLint

[error] Replace `····` with `··`

(prettier/prettier)